paying_for_college/models.py

```diff
@@ -400,8 +400,10 @@ def total_fields(field_list):
     def cost_error(self):
         """Return 1 or 0: Is total-cost less than tuition?"""
         url_data = self.parsed_url
-        if url_data and url_data['totl'] != '' and url_data['tuit'] != '':
-            if int(url_data['totl']) < int(url_data['tuit']):
+        if url_data:
+            totl = url_data.get('totl') or 0
+            tuit = url_data.get('tuit') or 0
```

Comment:
Not nitpicky. I started with `url_data.get('totl', 0)` but that would allow allow an empty string to pass through, and I actually want to coerce both empty string and None to zero, and `or` does that.